### ISSUE TITLE
Visualize: 2d Ideals don't render Mac Mojave (Safari or Opera)

### ISSUE THREAD

**Issue by aarondall** [Macaulay2/M2#876]
The following code is taken directly from the documentation for the Visualize package.

```
loadPackage "Visualize";
S = QQ[x,y];
I = ideal"x4,xy3,y5";
visualize I
```

The default browser opens to a page that is blank save for the **Menu** column on the right hand side. The 2d staircase doesn't render. Switching default browsers didn't help.

![screen shot 2018-11-03 at 4 00 41 pm](https://user-images.githubusercontent.com/6551646/47953710-a366d200-df81-11e8-99ff-7ad14f0a3ff8.jpg)

On the other hand, 

```
R = QQ[x,y,z]
J = ideal"x4,xyz3,yz2,xz3,z6,y5"
visualize J
```
produces the desired 3d staircase. 

![screen shot 2018-11-03 at 4 01 45 pm](https://user-images.githubusercontent.com/6551646/47953733-d14c1680-df81-11e8-8079-db2bdf9b1161.jpg)

**Comment by b-stone** [comment]
This is interesting. Does it work in Chrome on your machine?

**Comment by aarondall** [comment]
No. It gives the same empty webpage as in the screenshot above.

**Comment by DanGrayson** [comment]
This is still a problem.

**Comment by b-stone** [comment]
I have fixed this my pull request #951. Sorry it took so long.
